android/app/src/main/res/values-ro/strings.xml

```diff
@@ -1918,4 +1921,8 @@
 	<string name="type.amenity.events_venue">Locul de desfășurare a evenimentelor</string>
 	<string name="type.shop.auction">Licitație</string>
 	<string name="type.shop.collector">Colecții</string>
+	<string name="type.self_service.yes">Self-service disponibil</string>
```

Comment:
This is consistent with the other translations below:
```suggestion
	<string name="type.self_service.yes">Autoservire disponibilă</string>
```

Reply:
Done ^^.